code/lib/cli/src/sandbox-templates.ts

```diff
@@ -154,6 +163,7 @@ const baseTemplates = {
       builder: '@storybook/builder-webpack5',
     },
     skipTasks: ['e2e-tests-dev', 'bench'],
+    projectType: ProjectType.WEBPACK_REACT,
```

Comment:
Are all `ProjectType` related changes needed just to skip something based on REACT_NATIVE? Because if so, you don't need any of that, given that the sandbox command reads from sandbox-templates, and it does not have react-native

Reply:
No. It's also needed later to start `dev`, since we have to set the flags for the onboarding, if it is a React project